reproject/hips/core.py

```diff
@@ -409,8 +409,14 @@ def coadd_hips(input_directories, output_directory):
                         raise NotImplementedError(
                             "Convert jpg to png to allow for blending/coadding"
                         )
+                    elif tile_format == "fits":
+                        header = fits.getheader(filepath)
+                        image1 = fits.getdata(filepath)
+                        image2 = fits.getdata(target_filepath)
+                        result = image1 + image2
```

Comment:
The problem with this is that as you get to the lower res tiles you will keep summing the fluxes so the effective flux will be much higher in the low res tiles. I think you want to take an average of the two here.

Reply:
Aha, makes sense!